docs/password_hashing.rst

```diff
@@ -50,7 +50,10 @@ Password storage and verification
 
 All implementations of the modular crypt hasher `str` function
 internally generate a random salt, and return a hash encoded
-in ascii modular crypt format, which can be stored in a shadow-like file::
+in ascii modular crypt format, which can be stored in a shadow-like file
+
+.. doctest::
+    :pyversion: > 3.3
```

Comment:
What does this directive do? It's new to me 😄

Reply:
This is a very nice feature in sphinx which, since version 1.6, lets you skip running the tests if the runtime version doesn't match the specified condition